libs-scala/gatling-utils/src/main/scala/com/daml/gatling/stats/SchedulerStats.scala

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.gatling.stats
+
+import com.daml.gatling.stats.SimulationLog.RequestTypeStats
+import scalaz.\/
+import scalaz.std.list._
+import scalaz.syntax.either._
+import scalaz.syntax.std.option._
+import scalaz.syntax.traverse._
+
+case class SchedulerStats(
+    sync: Count[Int],
+    accp: Option[Count[Int]] = None,
+    comp: Option[Count[Int]] = None
+) extends Formattable {
+  def formatted: String =
+    List(
+      "=" * lineLength,
+      subtitle("Scheduler message"),
+      msgLine("Acknowledgement", sync.some),
+      msgLine("ACCEPTED message", accp),
+      msgLine("COMPLETED message", comp),
+      "=" * lineLength
+    ).mkString("\n")
```

Comment:
Again a very specific use case here, better replace this, too. (and the test data)